examples/cahnHilliard/sphereDaemon.py

```diff
@@ -10,7 +10,7 @@
 class SphereDaemon(MayaviDaemon):
 
     def view_data(self):
-        """Sets up the mayavi pipeline for the visualization.
```

Comment:
[mayavi](https://github.com/enthought/mayavi) used to be [MayaVi](http://mayavi.sourceforge.net/), but now it's [mayavi](https://github.com/enthought/mayavi), except when sometimes it's [Mayavi](https://github.com/enthought/mayavi/blob/master/README.rst#installation). *le sigh*